data/map.txt

```diff
@@ -31142,6 +35454,15 @@ planet "Ring of Wisdom"
 	bribe 0
 	security 0.8
 
+planet Ruelogakk
+	attributes "gegno quarg"
+	landscape land/station5
+	description `This Quarg station, although pristine and kept in high order, has been constructed rather recently compared to the complete ringworld the next system over. It appears to be purposed as some sort of resting spot for tourists, though very few docking bays are in use. A hauntingly low amount of Quarg are seen walking about the brightly lit abstract hallways of glass with only the faintest hum of machinery.`
+	spaceport `Only a few small Gegno transports can be seen docked with this station, hooked up to strange devices that seem to match the designs of their ships. Other than that, the Gegno's presence here seems to be kept at an extreme minimum, with at most one or two seen passing by occasionally.`
```

Comment:
Notably, the other two Quarg location desriptions don't actually name the Gegno. Should we remove their name from this one as well?